core/src/main/java/google/registry/flows/certs/CertificateChecker.java

```diff
@@ -162,7 +158,7 @@ public ImmutableSet<CertificateViolation> checkCertificate(X509Certificate certi
     ImmutableSet.Builder<CertificateViolation> violations = new ImmutableSet.Builder<>();
 
     // Check if currently in validity period
-    Date now = clock.nowUtc().toDate();
+    DateTime now = clock.nowUtc();
```

Comment:
This is a functionality change?  Are you sure this is even correct?

What's the motivation for removing `Date` everywhere anyway?  Sometimes you do legitimately need to process just a date, not a date and an attached time.